### ISSUE TITLE
Drawing candles?

### ISSUE THREAD

**Issue by Oleg-MBO** [wcharczuk/go-chart#97]
Is it possible to draw candles using this library?
https://www.investopedia.com/trading/candlestick-charting-what-is-it/

**Comment by anastalaz** [comment]
I don't think you can draw candlesticks with this library. The only thing I found was a line chart in the stock analysis example. Searching on github I found [this](https://github.com/carlesar/go-candles) which looks promising. The candles are not perfectly aligned but it is a start.

**Comment by jokaorgua** [comment]
@wcharczuk do you plan to add candlesticks to your charting lib?
